test/black-box/itemcategory/ItemCategoryRemoveCommand.test.ts

```diff
@@ -138,6 +145,7 @@ describe('ItemCategoryRemoveCommand', () => {
         const res = await testUtil.rpc(categoryCommand, [categoryRemoveCommand, createdCategory.id]);
```

Comment:
when possible, use testUtil.generateData() instead of addData(). Usage of addData creates much more work and errors when the models are changed.